roles/test_operator/defaults/main.yml

```diff
@@ -100,36 +103,36 @@ cifmw_test_operator_tempest_config:
     name: "{{ test_operator_job_name }}"
     namespace: "{{ cifmw_test_operator_namespace }}"
   spec:
-    containerImage: "{{ cifmw_test_operator_tempest_image }}:{{ cifmw_test_operator_tempest_image_tag }}"
-    storageClass: "{{ cifmw_test_operator_storage_class }}"
-    privileged: "{{ cifmw_test_operator_privileged }}"
-    parallel: "{{ cifmw_test_operator_tempest_parallel | default(omit) }}"
-    SSHKeySecretName: "{{ cifmw_test_operator_tempest_ssh_key_secret_name | default(omit) }}"
-    configOverwrite: "{{ cifmw_test_operator_tempest_config_overwrite | default(omit) }}"
-    networkAttachments: "{{ cifmw_test_operator_tempest_network_attachments }}"
+    containerImage: "{{ _test_operator_tempest_image }}:{{ _test_operator_tempest_image_tag }}"
+    storageClass: "{{ _test_operator_storage_class }}"
+    privileged: "{{ _test_operator_privileged }}"
+    parallel: "{{ _test_operator_tempest_parallel | default(omit) }}"
+    SSHKeySecretName: "{{ _test_operator_tempest_ssh_key_secret_name | default(omit) }}"
+    configOverwrite: "{{ _test_operator_tempest_config_overwrite | default(omit) }}"
+    networkAttachments: "{{ _test_operator_tempest_network_attachments }}"
     tolerations: "{{ cifmw_test_operator_tolerations | default(omit) }}"
     nodeSelector: "{{ cifmw_test_operator_node_selector | default(omit) }}"
```

Comment:
Those vars are referenced prior to the stage testing in some tasks, I didn't want to change them to avoid making this configuration vars inconsistent across the role.

Reply:
Ok, this probably makes sense. As we've already discussed during our call we probably want to aim at this:

- variables with `cifmw_test_operator_[test-operator CR name]_[parameter name]`  name should be modifiable inside a stage.
- variables with `cifmw_test_operator_[parameter name]` should not be modifiable inside the stage

But in the long term the tolerations and nodeSelector should be moved to `cifmw_test_operator_[test-operator CR name]_[parameter name]`.

Reply:
Generally I am not against it for future but I would rather not making those changes inside this patch as this might have implications on other stages in the deployment of the operator itself which I believe we should look into before making this change.

Reply:
Yes this part should definitely be made in a separate PR:)

> But in the long term the tolerations and nodeSelector should be moved to cifmw_test_operator_[test-operator CR name]_[parameter name].